### PR TITLE
ARUHA-906: Rewrite readme

### PR DESCRIPTION
**Pull request by SergKam**
ARUHA-906: Rewrite readme
https://techjira.zalando.net/browse/ARUHA-906

## Description
Readme re-writed according to https://github.com/zalando/zalando-howto-open-source

## Review
- [ ] Documentation
- [ ] CHANGELOG

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,858 +2,268 @@
...
-
-The project also provides compatability with the [STUPS project](https://stups.io/). Additional features that we plan to cover in the future are:
-
-* Discoverability of the resource structures flowing into the broker.
-
-* A managed API that allows consumers to subscribe and have stream offsets stored by the server.
+## [Nakadi Event Broker](https://zalando.github.io/nakadi/)
 
-* Filtering of events for subscribing consumers.
+The goal of Nakadi (**ნაკადი** means "stream" in Georgian) is to provide an event broker infrastructure to:
```
Comment (lmontrieux) [inline README.md]: This is the `goal`, but.. what *is* Nakadi?

File: README.md [hunk 2]
```diff
@@ -2,858 +2,268 @@
...
 
-* Role based access control to data. 
+- Abstract event delivery via a secured [RESTful API](https://zalando.github.io/nakadi/manual.html#nakadi-event-bus-api).
+ 
+    This allows microservices teams to maintain service boundaries, and not directly depend on any specific message broker technology.
+    Access can be managed individually for every Event Type and secured using *OAuth* and custom authorization plugins.
```
Comment (lmontrieux) [inline README.md]: Forward reference: mentions Event Type before defining it

File: README.md [hunk 3]
```diff
@@ -2,858 +2,268 @@
...
 
-* Support for different streaming technologies and engines. Nakadi currently uses [Apache Kafka](http://kafka.apache.org/) as its broker, but other providers (such as Kinesis) will be possible. 
+- Enable convenient development of event-driven applications and asynchronous microservices. 
 
-More detailed information can be found on the [manual](http://zalando.github.io/nakadi-manual/). 
+    Event types can be defined with [Event type schemas](https://zalando.github.io/nakadi/manual.html#using_event-types) 
+    and managed via a registry. All events will be validated against the schema before publishing the event type. 
+    It allows to granite the data quality and data consistency for the data consumers.
```
Comment (lmontrieux) [inline README.md]: granite?

File: README.md [hunk 4]
```diff
@@ -2,858 +2,268 @@
...
+     
+- Efficient low latency event delivery. 
+    
+    Once a publisher sends an event using a simple [HTTP POST](https://zalando.github.io/nakadi/manual.html#using_producing-events), 
+    consumers can be pushed to via a [streaming](https://zalando.github.io/nakadi/manual.html#using_consuming-events-lola)
+    HTTP connection, allowing near real-time event processing. 
+    The consumer connection has keepalive controls and support for managing stream offsets using
+    [subscriptions](https://zalando.github.io/nakadi/manual.html#using_consuming-events-hila). 
+
+More detailed information can be found on our [Website](http://zalando.github.io/nakadi/).
```
Comment (lmontrieux) [inline README.md]: website (lower case)

File: docs/_documentation/architecture_timelines.md [hunk 5]
```diff
@@ -3,14 +3,14 @@ title: Timelines
 position: 101
 ---
 
-Timelines
----------
+## Timelines
```
Comment (lmontrieux) [inline docs/_documentation/architecture_timelines.md]: Shouldn't this be one level higher than the others?

Comment (SergKam) [inline docs/_documentation/architecture_timelines.md]: This should be ##. I moved all other one level deeper.

**Comment by v-stepanov** [comment]
👍

**Comment by SergKam** [comment]
:+1:

**Pull request merged**